src/lesson3/SingleArray.java

```diff
@@ -0,0 +1,160 @@
+package lesson3;
+
+
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+
+public class SingleArray {
+
+
+    public static void searchInArray() {
+        int[] array = new int[10];
+        Random r = new Random();
+        for (int i = 0; i < array.length; i++) {
+            array[i] = r.nextInt();
+        }
+        Scanner scan = new Scanner(System.in);
+        System.out.println(Arrays.toString(array));
+        System.out.println("Введите число, которое хотите найти в массиве: ");
+        int input = scan.nextInt();
+        boolean flag = false;
+        for (int j : array) {
+            if (j == input) {
+                flag = true;
+                break;
+            }
+        }
+        if (flag) {
+            System.out.println("Ваше число найдено в массиве!");
+        } else {
+            System.out.println("Ваше число не найдено в массиве!");
+        }
+    }
+
+    public static void removeInArray() {
+        int[] array = new int[20];
+        Random r = new Random();
+        for (int i = 0; i < array.length; i++) {
+            array[i] = r.nextInt();
+        }
+        Scanner scan = new Scanner(System.in);
+        System.out.println(Arrays.toString(array));
+        System.out.println("Введите число, которое хотите удалить в массиве: ");
+        int input = scan.nextInt();
+        boolean found = false;
```

Comment:
Maybe, if you have `countInput`, which determines, whether we have necessary number, we can remove this flag?

Reply:
my bad